tools/aot/BUILD

```diff
@@ -1,3 +1,8 @@
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Also available under a BSD-style license. See LICENSE.
+
```

Comment:
Is this license common for repos in `cruise-automation`?  Do we need to get any permission?

Reply:
This is the legal approved header and in use in most of our source files. Adding it here as it was missed initially.